backend/tracim_backend/lib/core/event.py

```diff
@@ -162,16 +162,6 @@ def get_paginated_messages_for_user(
         ).order_by(Message.event_id.desc())
         return get_page(query, per_page=count, page=page_token or False)
 
-    def get_messages_count(
-        self,
-        user_id: int,
-        read_status: ReadStatus,
-        event_types: typing.List[EventTypeDatabaseParameters] = None,
-    ) -> int:
-        return self._base_query(
-            user_id=user_id, event_types=event_types, read_status=read_status
-        ).count()
-
```

Comment:
Isn't this method used for an API (https://github.com/tracim/tracim/pull/3394)?

Reply:
Yes, this is a mistake, I'll revert this